lib/db/scope.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const unbuf = require('buf').unbuf.hex;
```

Comment:
let's add a comment above here to explain what this file is for. 
Example: `This module consists of helpers that are used by both memory and MySQL database engines`